builder/default.nix

```diff
@@ -1,91 +1,35 @@
-{ pkgs, buildPackages, stdenv, lib, haskellLib, ghc, buildGHC, fetchurl, writeText, runCommand, pkgconfig, nonReinstallablePkgs, ghcForComponent, hsPkgs }:
-
-{ flags
-, package
-, components
-, cabal-generator
-
-, name
-, sha256
-, src
-, revision
-, revisionSha256
-, patches
-
-, preUnpack
-, postUnpack
-, preConfigure
-, postConfigure
-, preBuild
-, postBuild
-, preCheck
-, postCheck
-, preInstall
-, postInstall
-, preHaddock
-, postHaddock
-
-, shellHook
```

Comment:
Deleting shellHook from the comp-builder.nix input will break my (already merged) changes for configurable shellHooks per-package: #117

Reply:
This code has been moved not deleted. You should find shellHook in `hspkg-builder.nix`.